docs/src/content/docs/book/maps.mdx

```diff
@@ -151,6 +153,29 @@ if (fizz == null) {
 }
 ```
 
+### Compare with `.deepEquals()` {#deepequals}
+
+<Badge text="Available since Tact 1.5" variant="tip" size="medium"/><p/>
+
+The `.deepEquals(){:tact}` [method](/book/functions#extension-function) on maps returns `true{:tact}` if all the entries of the map match respective entries of another map and their [underlying serialization logic](https://docs.ton.org/develop/data-formats/tl-b-types#hashmap) is exactly the same. Returns `false{:tact}` otherwise.
```

Comment:
Actually, it's the other way around: it returns true even if the serializations are different (as opposed to equality comparisons via hashes). This is called extensional equality. You don't have to explain everything at the level of TL-B, but an example (from the corresponding tests) is definitely needed here to explain the use case of deepEquals. We can also add a note, that if all maps are formed in a controlled way the hash comparisons are fine, but if a map comes from an external sources that does not provide those guarantees, one needs to use deepEquals

Reply:
Addressed the suggestions :)